### DISCUSSION TITLE
Changing fullscreen orientation inside player

### DISCUSSION THREAD

**Discussion by KasraIDK** [libre-tube/LibreTube#5824] [Ideas]
### Describe your suggested feature

Two possible methods I've thought of:
1. Auto rotate to all orientations by default with a rotation lock toggle
2. A button similar to what already exists in Settings > Player > Fullscreen orientation

### Other details

Having this option be more easily accessible inside the player makes sense and is much nicer than going through settings just to change one option each time. 

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] I will fill out all of the requested information in this form.

**Comment by KasraIDK** [comment]
Here are some (badly made 🙃) mockups
![20240306_184052](https://github.com/libre-tube/LibreTube/assets/41045880/2c720149-deee-4f59-a180-53de6d0ff35a)
![20240306_184131](https://github.com/libre-tube/LibreTube/assets/41045880/d7b8002f-882e-4864-a4cb-9b6871b3ab1f)

**Comment by Bnyro** [comment]
As of your opinion, should we also remember the previous orientation selection and remove the preference for it? Or should changing the orientation at the player only be temporary and be resetted again after the video has finished?

**Comment by KasraIDK** [comment]
I think the current option in settings should be kept and treated as a default, and the option in the player would be a temporary override that resets to default after opening a new video.

As an example VLC does this but they also give the option to remember last locked rotation (not a necessary option to have imo).

I believe this combined with the rotation lock button (first picture) would be a good implementation.

**Reply by Bnyro** [reply]
In my opinion the player interface and the player options bottom sheet are quite cluttered already, so I'm not yet really sure about such an option.

I'd like to wait for some people interested in that feature first.

**Comment by ghost** [comment]
I would appreciate if the lock button on the upper left simply locked the orientation, too. When laying on my side it is super annoying that the app switches back to portrait orientation. I always watch videos in fullscreen but use portrait orientation for everything else.
